.github/workflows/pr-vrt-comment.yml

```diff
@@ -1,15 +1,10 @@
-name: VRT CI | Comment on PR
```

Comment:
lets keep the name, dont wanna introduce additional pipeline within actions list - we can change once everything is tested and working

Reply:
Reverted to old workflow names